frontend/src/metabase/css/components/popover.css

```diff
@@ -26,6 +26,7 @@
 	font-weight: bold;
 	background-color: rgb(76,71,71);
 	border: none;
+	padding: 0.8em;
```

Comment:
This is going to add extra padding to every existing tooltip. Please remove it and wrap `<Value>` in `<div className="p1">` (or maybe `p2`, see how it looks)